### ISSUE TITLE
typo- semicolon 

### ISSUE THREAD

**Issue by keithchugg** [xiph/opus#92]
the following line has a comma instead of a semicolon at the end which looks unintended:

https://github.com/xiph/opus/blob/1827db6a601e02729475c4618500f49f8b0cbefa/celt/celt_lpc.c#L110

**Comment by jmvalin** [comment]
Indeed, that should be a semicolon. It's harmless in practice, but I'll fix it. Thanks for reporting that!
